airflow/models/dagbag.py

```diff
@@ -124,7 +123,10 @@ def get_dag(self, dag_id, from_file_only=False):
         :param from_file_only: returns a DAG loaded from file.
         :type from_file_only: bool
         """
-        from airflow.models.dag import DagModel  # Avoid circular import
+        # Avoid circular import
+        from airflow.models.dag import DagModel
+        from airflow.models.serialized_dag import SerializedDagModel
```

Comment:
The reason why this was mainly added in CONDITIONALS is we don't need to import these modules unless `if self.store_serialized_dags is TRUE`.

Reply:
This change will load those classes even if `store_serialized_dags=False` which is the default behavior

Reply:
Ah. Thanks. And obviously I have not read the comment even if it was there :(. Bad me.

Reply:
Comment added.